src/test/java/com/string/ReverseWordsTest.java

```diff
@@ -0,0 +1,15 @@
+package com.string;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ReveresWordsTest {
+
+    @Test
+    void testAlphabetical() {
+        ReverseWords reverseWords = new ReverseWords();
```

Comment:
Good to see that you implemented test structure properly. But I'm not sure that this test will be passing, as you used `returnReverseWords()` directly without providing proper info to compiler about where this method will be present. Also in your case, as you have written 
```java
public static String returnReverseWords(String s) {...}
``` 
, you actually don't need an object to access static method. 
You can do following in test class
```java
Assertions.assertEquals(true, ReverseWords.returnReverseWords("this is my car"), "siht si ym rac");
```

Or you could remove `static` from method declaration and use object to access method like 
```java
Assertions.assertEquals(true, reverseWords.returnReverseWords("this is my car"), "siht si ym rac");
```

Please check these and do needful.

Reply:
I changed `testAlphabetical` to `testReverseWords`
and removed `static` keyword from `returnReverseWords`

After pushing these changes I am not able to pass 1 check which is `Java gradle CI / Test algorithms (macos-latest, 1.8) (pull_request)`. I don't know how to work with that.

Reply:
Hi @loveshdongre , I'm not sure of the issue, I will let you know if I find anything. If you find anything, please update over here so anyone else checking this issue can focus on something else.

Reply:
Yes sure

Reply:
Hey @ayaankhan98 , if you have some spare time, please check this issue 

> After pushing these changes I am not able to pass 1 check which is `Java gradle CI / Test algorithms (macos-latest, 1.8) (pull_request)`. I don't know how to work with that. Need help!

I'm also looking into but as I'm unable to view logs, I can't pinpoint exact error. But seeing console logs, I think issue is related to OpenGL with macOS (although only jdk8 is failing and not jdk11, so I can be wrong).  Thanks :)

Reply:
I changed `package strings;` to `package com.strings;` and now everything works. my bad.

Reply:
Yeah, I missed that too, good work.

Reply:
Should I close the pull request and wait for it to get merged?